### DISCUSSION TITLE
Toggle follow_cursor

### DISCUSSION THREAD

**Discussion by mihaifm** [hedyhli/outline.nvim#67] [Help] [answered]
Hi and thanks for the plugin.

Is there a way to stop following cursor, once it's enabled via :OutlineFollow ?

**Comment by hedyhli** [comment, marked as answer]
Hi, `:OutlineFollow` simply sets the currently focused symbol in the outline to the symbol you are currently on, in the code.

The automatic cursor following (which is triggered manually through `:OutlineFollow`) is determined by the config option `outline_items.auto_update_events.follow`. See [the default config options](https://github.com/hedyhli/outline.nvim?tab=readme-ov-file#default-options).

**Comment by hedyhli** [comment]
Please reopen if it didn't work. Thanks!
